templates/education/pdce-workshops.php

```diff
@@ -11,16 +11,21 @@
 
 $the_title          = get_the_title();
 
-// Some custom fields
-$coursecodeundergrad			= get_post_meta( absint( $post_id ), 'coursecodeundergrad', true );
-$coursecodegrad					= get_post_meta( absint( $post_id ), 'coursecodegrad', true );
-$vanityurl 						= get_post_meta( absint( $post_id ), 'vanityurl', true );
-$credit 						= get_post_meta( absint( $post_id ), 'credit', true );
-$promophoto 					= get_post_meta( absint( $post_id ), 'promophoto', true );
-$subtitle						= get_post_meta( absint( $post_id ), 'subtitle', true );
-$dates							= get_post_meta( absint( $post_id ), 'dates', true );
-$location						= get_post_meta( absint( $post_id ), 'location', true );
+//Get the post permalink
+
+$permalink          = get_permalink( $post );
```

Comment:
You absolutely don't need to, I was just checking it wasn't a typo (as from what I can tell it's the only time the `$post` object is used). My only concern would be that it kinda appears like the `$post` variable has appeared out of nowhere. (Whilst you and I know it hasn't, it might be odd for a newcomer)